tests/test_humandetector.py

```diff
@@ -0,0 +1,293 @@
+        detection = self.detector.detect(images=[VLIMAGE_ONE_FACE])[0][0]
+        self.assertBoundingBox(detection.boundingBox)
+
+    def test_bounding_box_as_dict(self):
+        """
+        Test conversion bounding box to dictionary
+        """
+        boundingBox = self.detector.detectOne(image=VLIMAGE_ONE_FACE).boundingBox.asDict()
+
+        assert (
+            jsonValidator(schema=REQUIRED_HUMAN_BODY_DETECTION).validate(boundingBox) is None
+        ), f"{boundingBox} does not match with schema {REQUIRED_HUMAN_BODY_DETECTION}"
+
+    def test_human_detection_as_dict(self):
+        """
+        Test conversion result human detection to dictionary
+        """
+        for case in self.landmarksCases:
+            with self.subTest(landmarks5=case.detectLandmarks):
+                detectAsDict = self.detector.detectOne(
+                    image=VLIMAGE_ONE_FACE, detectLandmarks=case.detectLandmarks
+                ).asDict()
+                assert (
+                    jsonValidator(schema=REQUIRED_HUMAN_BODY_DETECTION).validate(detectAsDict) is None
+                ), f"{detectAsDict} does not match with schema {REQUIRED_HUMAN_BODY_DETECTION}"
+
+    def test_detection_with_default_detector_type(self):
+        """
+        Test human detection with default detector type
+        """
+        for detectionFunction in ("detect", "detectOne"):
+            with self.subTest(detectionFunction=detectionFunction):
+                if detectionFunction == "detectOne":
+                    detection = self.detector.detectOne(image=VLIMAGE_ONE_FACE)
+                else:
+                    detection = self.detector.detect(images=[VLIMAGE_ONE_FACE])[0]
+                self.assertHumanDetection(detection, VLIMAGE_ONE_FACE)
+
+    def test_batch_detect_using_different_type_detector(self):
+        """
+        Test batch detection using different type of detector
+        """
+        detection = self.detector.detect(images=[VLIMAGE_ONE_FACE])[0]
+        self.assertHumanDetection(detection, VLIMAGE_ONE_FACE)
+
+    def test_detect_one_with_image_of_several_humans(self):
+        """
+        Test detection of one human with image of several humans
+        """
+
+        detection = self.detector.detectOne(image=VLIMAGE_SEVERAL_FACE)
+        self.assertHumanDetection(detection, VLIMAGE_SEVERAL_FACE)
+
+    def test_detect_one_with_image_without_humans(self):
+        """
+        Test detection of one human with image without humans
+        """
+        imageWithoutFace = VLImage.load(filename=NO_FACES)
+
+        detection = self.detector.detectOne(image=imageWithoutFace)
+        assert detection is None, detection
+
+    def test_batch_detect_with_image_without_humans(self):
+        """
+        Test batch human detection with image without humans
+        """
+        imageWithoutFace = VLImage.load(filename=NO_FACES)
+        detection = self.detector.detect(images=[imageWithoutFace])
+        assert 0 == len(detection[0])
```

Comment:
excepted 0, got len(detection[0])

Reply:
trace for failed test:
```
>               assert 20 == len(detection[0])
E               AssertionError: assert 20 == 0
```
I think that is enough for debuging

Reply:
only for top level, for detection[0][N] and deeper - that isn't enought

Reply:
i don't undestand you. detection list is empty for the image